### ISSUE TITLE
[BUG] rxjs/finnish throws false errors with combineLatest

### ISSUE THREAD

**Issue by distante** [cartant/eslint-plugin-rxjs#115]
Plugin version: 5.0.3
rxjs Version: 7.8.1

[combineLatest](https://rxjs.dev/api/index/function/combineLatest) can also take an object as key/observable that makes easier to  maintain the observable since the order is not important anymore when spreading the values.

So instead of

```ts
combineLatest([obs1$, obs2$]).pipe(
    map(([obs1, obs2]) => {
     // do something
    })
)
```

we can do

```ts
combineLatest({
  obs1: obs1$,
  obs2: obs2$
}).pipe(
    map(({obs2, obs1}) => { // note that the order is backwards, but it is ok here.
     // do something
    })
)
```

The problem is that rxjs/finnish will detect ` obs1: obs1$,` and `obs2: obs2$` as an error with `Finnish notation should be used here.`

**Comment by JasonWeinzierl** [comment]
This is fixed in forked package `eslint-plugin-rxjs-x` v0.7.4
